spec/person_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'yaml'
+require './lib/person.rb'
+
+
+person = YAML.load_file('./lib/person.yml')
+describe Person do
```

Comment:
You have multiple files that test the `Person` class. Put all of the tests in the same file for greater readability.